src/__tests__/App.test.js

```diff
@@ -0,0 +1,15 @@
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import App from '../App';
+import store from '../Redux/configurStore';
+
+describe('test App', () => {
+  it('test snapshoot of app', () => {
+    const tree = render(
+      <Provider store={store}>
+        <App />
+      </Provider>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
- [ ] Dear @girma336, Good job adding snapshot tests for your components, But to make our test more reliable we need to add a test for some pure functions in the Redux code. 
- [ ] Also, we need to have an integration test using React Testing Library, this is to test the user interaction so that we can assert the user experience for our app. Please add these tests to make your app more reliable.